### ISSUE TITLE
The Web Version of Stellarium DOES NOT WORK PROPERLY!!! 

### ISSUE THREAD

**Issue by MichaelIsGreat123** [Stellarium/stellarium-web-engine#20]
Hello the Stellarium team,

It has been really extremely discouraging to see that I contacted a guy named Fabian SEVERAL TIMES EONS AGO AS WELL AS RECENTLY to sort out the problems that are present in the web version of Stellarium and STILL NOTHING HAS BEEN CORRECTED to correct these problems. Here is what I sent.

The web version still does not work properly. It does not even save the location when I select it.

See my posts that have still not been solved at all and could you let me know if you intend to solve them in the future?
---"3 points to sort out in the web version of Stellarium. Thanks. · Issue Stellarium/stellarium#14 · Stellarium/stellarium-web · GitHub" at https://github.com/Stellarium/stellarium-web/issues/14

---"In Firefox, when I update the date and time and location, it simply does not save them. · Issue Stellarium/stellarium#12 · Stellarium/stellarium-web" at https://github.com/Stellarium/stellarium-web/issues/12

Thanks to let me know if these problems, you intend to have them solved for good one day? OR NOT?!!

Highly discouraging to see that problems are not corrected. I know, it is free and so on but it is better to not make available something that will never be corrected when needed.

Kind Regards
Michael

### Expected Behaviour: 
I click on unknown to save my location, move the pointer to my location and it keeps being saved the next time I open the web version of Stellarium.
Same for every setting that I have selected at a previous session of Firefox.

### Actual Behaviour
Using Firefox latest version (70.0):
When I click on unknown to save my location, I move the pointer to an accurate location, BUT it is NOT saved for good each time I reopen Firefox. Same for the options I chose. Same the time is not correct, it is NOT the time I have on my computer!! It is day light and I have the web version of Stellarium showing that it is night!!!

### Steps to reproduce:
See above.

### System
* Stellarium version: The web version of Stellarium
* Operating system: Windows 10 Pro, latest update.
Firefox latest version (70.0).
* Graphics Card: built-in Intel core i7 9700K 4K graphics card.
### Logfile 
I use only the web version of Stellarium so I do not think that there is a log file.

**Comment by gzotti** [comment]
There is always the option to NOT use free software that does not work for you.

**Comment by MichaelIsGreat123** [comment]
Hello Georg,


Proud of your reply, Georg?!!! Honestly, what is wrong with you and that you cannot even see that your reply is wrong is even worse!!
And yes, I repeat especially for you Georg:
If I would make a software available, I would do that only if I am confident that I can correct any issues that exist with this software. That is basic common sense to me and it should be to you too.
If the software does not work for me (the web version of Stellarium), it certainly is not working for anyone using it!!

As you can see, and I am going to put the copy of the email below, I contacted Fabien eons ago and even recently and I did not receive any feedback that one day (hopefully soon), the problems with the web  version of Stellarium will be corrected for good.

I was expecting the following reply:
Thanks for your feedback. We will try to make the web version of Stellarium work, hopefully in the coming months.


Kind Regards
Michael
michaeljustgreat1@hotmail.com



--------------------------------------------------------

Hello Fabian,


The web version still does not work properly. It does not even save the location when I select it.


See my posts that have still not been solved at all and could you let me know if you intend to solve them in the future?
---"3 points to sort out in the web version of Stellarium. Thanks. · Issue #14 · Stellarium/stellarium-web · GitHub" at https://github.com/Stellarium/stellarium-web/issues/14

---"In Firefox, when I update the date and time and location, it simply does not save them. · Issue #12 · Stellarium/stellarium-web" at https://github.com/Stellarium/stellarium-web/issues/12

Thanks to let me know if these problems, you intend to have them solved for good?


Kind Regards
Michael

---------------------------------------------------
________________________________

**Comment by gzotti** [comment]
Then please why don't you show better performance and make some software available to the world that works 100%?

The facts that it works for me on 8 desktops, notebooks, tablets and phones, and that we don't have received thousands of similar rants, speaks for itself. Your aggressive tone is certainly not helpful. Again, if it does not work for you, just don't bother using it. 

BTW you have received answers to your questions at https://github.com/Stellarium/stellarium-web/issues/12.

**Comment by xalioth** [comment]
Hi Michael,

as Georg pointed out, I already replied to most of your questions at https://github.com/Stellarium/stellarium-web/issues/12. As it's currently not possible to save your location on the website (for reasons I explained to you already), if you really want to use your own location I'd suggest that you allow your browser to share it with the website, and it will solve your issue.

Also, for the time set at startup: I chose to automatically set the start time on the evening of the current day for your location. This is to avoid users to start the website and see only the daylight, which confused many of them. I chose this to be the default behavior for everyone. I am aware that such decisions will not suit some users, but until we have a proper way to save settings it will be the only way.

Fabien
